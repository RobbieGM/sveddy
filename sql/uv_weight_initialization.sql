DROP SCHEMA public CASCADE;
CREATE SCHEMA public;
GRANT ALL ON SCHEMA public TO current_user;
GRANT ALL ON SCHEMA public TO public;
CREATE EXTENSION sveddy;

SELECT cardinality(arr) from get_initial_weights_uv(8) as arr;
