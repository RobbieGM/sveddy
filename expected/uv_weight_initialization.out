DROP SCHEMA public CASCADE;
NOTICE:  drop cascades to 4 other objects
DETAIL:  drop cascades to extension sveddy
drop cascades to table ratings
drop cascades to table ratings_sveddy_model_u
drop cascades to table ratings_sveddy_model_v
CREATE SCHEMA public;
GRANT ALL ON SCHEMA public TO current_user;
GRANT ALL ON SCHEMA public TO public;
CREATE EXTENSION sveddy;
SELECT cardinality(arr) from get_initial_weights(8) as arr;
 cardinality 
-------------
           8
(1 row)

